nixos/modules/services/web-apps/nextjs-ollama-llm-ui.nix

```diff
@@ -0,0 +1,87 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.nextjs-ollama-llm-ui;
+  # we have to override the URL to a Ollama service here, because it gets baked into the web app.
+  nextjs-ollama-llm-ui = cfg.package.override { ollamaUrl = "https://ollama.lambdablob.com"; };
```

Comment:
Shouldn't this be using the user's chosen `ollamaUrl`?

I can't get it working unless I override the package manually.

Reply:
Yep, this slipped through. It will be fixed with the next release https://github.com/NixOS/nixpkgs/pull/315184